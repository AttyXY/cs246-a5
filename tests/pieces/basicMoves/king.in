setup
+ k c4
+ K f5 
+ R a1
done
game human human
move f5 e4
move c4 c5
move e4 f5
move c5 c4
move f5 g5
move c4 b4
quit
quit
