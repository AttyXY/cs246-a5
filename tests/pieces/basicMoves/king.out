WELCOME TO CHESS! PLEASE ENTER ONE OF THE FOLLOWING COMMANDS TO BEGIN:
- setup
- game [player1] [player2]

STARTING SETUP

8  --------
7  --------
6  --------
5  --------
4  --k-----
3  --------
2  --------
1  --------
   abcdefgh

8  --------
7  --------
6  --------
5  -----K--
4  --k-----
3  --------
2  --------
1  --------
   abcdefgh

8  --------
7  --------
6  --------
5  -----K--
4  --k-----
3  --------
2  --------
1  R-------
   abcdefgh

STARTING GAME!

8  --------
7  --------
6  --------
5  -----K--
4  --k-----
3  --------
2  --------
1  R-------
   abcdefgh

8  --------
7  --------
6  --------
5  --------
4  --k-K---
3  --------
2  --------
1  R-------
   abcdefgh

8  --------
7  --------
6  --------
5  --k-----
4  ----K---
3  --------
2  --------
1  R-------
   abcdefgh

8  --------
7  --------
6  --------
5  --k--K--
4  --------
3  --------
2  --------
1  R-------
   abcdefgh

8  --------
7  --------
6  --------
5  -----K--
4  --k-----
3  --------
2  --------
1  R-------
   abcdefgh

8  --------
7  --------
6  --------
5  ------K-
4  --k-----
3  --------
2  --------
1  R-------
   abcdefgh

8  --------
7  --------
6  --------
5  ------K-
4  -k------
3  --------
2  --------
1  R-------
   abcdefgh

Final Score:
White: 0
Black: 0
THANKS FOR PLAYING CHESS!
